components/brave_new_tab_ui/components/default/editTopSite/style.ts

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+import styled from 'brave-ui/theme'
+import Button, { Props as ButtonProps } from 'brave-ui/components/buttonsIndicators/button'
+
+const isDarkTheme = (p: any) => {
+  return p.theme.name === 'Brave Dark'
+}
+
+interface StyledDialogWrapperProps {
+  textDirection: string
+}
+
+export const StyledDialogWrapper = styled<StyledDialogWrapperProps, 'div'>('div')`
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  z-index: 5;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`
+
+interface StyledDialogProps {
+  textDirection: string
+}
+
+export const StyledDialog = styled<StyledDialogProps, 'div'>('div')`
+  position: relative;
+  width: 437px;
+  top: -50px;
+  background-color: ${p => p.theme.color.contextMenuBackground};
+  border-radius: 4px;
+  padding: 32px 24px 24px 24px;
+  display: flex;
+  flex-direction: column;
+
+  box-shadow: 0px 4px 12px 0px rgba(0, 0, 0, 0.2);
+`
+
+export const DialogTitle = styled<{}, 'div'>('div')`
+  font-weight: 600;
+  font-size: 15px;
+  line-height: 20px;
+  color: ${p => isDarkTheme(p) ? '#F0F2FF' : '#212529'};
+`
+
+export const StyledCloseIcon = styled<{}, 'div'>('div')`
```

Comment:
Please use a button, for accessibility reasons. You can use the similar button from the _Brave Together_ tooltip which has the correct element type, tooltip, and hover / active / focus styles, as well as animations. Feel free to copy if you don't want to move to a shared component, which would be extra effort, and we can make shared in a follow-up. Code is [here](https://github.com/brave/brave-core/blob/dd76f149491cc302f3eb2668ee2d9825a05ed6ab/components/brave_new_tab_ui/components/default/footer/togetherTooltip/index.tsx#L31-L36) and [here](https://github.com/brave/brave-core/blob/d216e8c964d9293942cfb505d154913b578003e9/components/brave_new_tab_ui/components/default/footer/togetherTooltip/style.ts#L85-L114).

![image](https://user-images.githubusercontent.com/741836/108823889-e2b3f500-7575-11eb-94e4-a255aeef2e8f.png)

Reply:
Copy & pasted. Let's make shared as a f/u.